index.html

```diff
@@ -6,6 +6,392 @@
 	<link rel="stylesheet" type="text/css" href="index.css">
 </head>
 <body>
-	
+
```

Comment:
As a standard, we indent our HTML 2 spaces (not tabs). Since this is the standard you're already working in, you can keep it.  But on future projects, I would recommend switching.